general/glossary.md

```diff
@@ -0,0 +1,52 @@
+# Glossary
+
+This documents aims at providing a common lexicon for terms used to refer to artifacts.
+
+### Motivation
+
+Throughout the documentation there are several terms which are often abused because we either have no alternative or there's not enough consensus on what's their canonical name.
+
+This leads to often ambiguous documentation because one term is used for different things or because we use different terms for the same artifact/object.
+
+If this document succeeds, we should end up having a documentation where each object/artifact has one single term to designate it. In other words, [Wittgenstein](https://en.wikipedia.org/wiki/Ludwig_Wittgenstein) should be proud of us (see [Tractatus Logico-Philosophicus](https://en.wikipedia.org/wiki/Tractatus_Logico-Philosophicus)).
```

Comment:
![picard-clap](https://user-images.githubusercontent.com/7074/94803082-0de05100-03e9-11eb-85e5-a692837f64ca.gif)

Reply:
I knew you would like it :heart_eyes:

Reply:
![nerdalert](https://user-images.githubusercontent.com/905006/94821613-0c6f5280-0402-11eb-96e1-2ba19b14a047.gif)